torchvision/ops/poolers.py

```diff
@@ -111,8 +111,7 @@ def _infer_scale(feature: Tensor, original_size: List[int]) -> float:
 def _setup_scales(
     features: List[Tensor], image_shapes: List[Tuple[int, int]], canonical_scale: int, canonical_level: int
 ) -> Tuple[List[float], LevelMapper]:
-    if not image_shapes:
-        raise ValueError("images list should not be empty")
+    assert len(image_shapes) != 0
```

Comment:
How come not the following?
```suggestion
    assert image_shapes
```

Reply:
I think I reverted the original asserts in most cases but yes you are right